retrievaltool/src/main/java/org/duracloud/retrieval/source/DuraStoreStitchingRetrievalSource.java

```diff
@@ -118,6 +121,23 @@ public void chunkStitched(String chunkId) {
             return stitcher.getContentFromManifest(item.getSpaceId(),
                                                    item.getContentId(),
                                                    fileStitcherListener);
+        } catch (DataSourceException dse) {
+            try {
+                if (contentStore.contentExists(item.getSpaceId(), item.getContentId())) {
+                    throw dse;
+                } else {
+                    StringBuilder msg = new StringBuilder();
+                    msg.append("The item does not exist in the space: ");
+                    msg.append(item);
+                    msg.append(dse.getMessage());
+
+                    log.error(msg.toString());
```

Comment:
This log statement isn't needed given the following throw of a MissingContentException (given that the code catching this exception creates a log entry)